sources/accessible-name-and-description.ts

```diff
@@ -492,29 +493,21 @@ export function computeTextAlternative(
 			}
 		}
 
-		if (
-			isHTMLInputElement(node) ||
-			isHTMLSelectElement(node) ||
-			isHTMLTextAreaElement(node)
```

Comment:
This was an incomplete check whether the `node` is labelable. However, `getLabels` already has this check baked in.